tools/java/org/unicode/cldr/json/LdmlConvertRules.java

```diff
@@ -125,6 +141,15 @@
         // in common/supplemental/windowsZones.xml
         "mapTimezones:mapZone:other",
 
+        // in common/supplemental/units.xml
+        "*:unitPreference:geq",
+        "*:unitPreference:skeleton",
+
+//        // TODO: Blocked by [CLDR-14272] - need more documentation
+//        // in common/supplemental/grammaticalFeatures.xml
+//        "grammaticalDerivations:deriveComponent:value0",
```

Comment:
Those are documented.

Reply:
those specific items are documented, yes. but the output file builds as a whole.